.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           token: ${{ secrets.AVALANCHE_PAT }}
       - uses: actions/setup-go@v3
         with:
-          go-version: '~1.19.12'
+          go-version: '~1.20.8'
```

Comment:
In general, this should be a separate PR rather than including within a larger PR.